device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/ProxiedSSLSocket.java

```diff
@@ -162,6 +162,7 @@ private void doTunnelHandshake(Socket tunnel, String host, int port) throws IOEx
     }
 
     @RequiredArgsConstructor
+    static
```

Comment:
Is it just the git showing this on new line?

Reply:
it's not not git, it actually was on a new line! :D